### ISSUE TITLE
Change slack join conversation to slack channel

### ISSUE THREAD

**Issue by sreyaguha1** [amyxzhang/policykit#142]
Not sure about this but thought it would be more consistent

**Comment by amyxzhang** [comment]
yeah it's called that bc that's the name of the Slack API. and the reason the Slack API calls it that is bc the API is for public channels, private channels, multiparty DMs, and 1-on-1 DMs. 

Before now, we've had a 1-to-1 mapping from action to API but I think maybe going forward it does make sense to make actions the way people would think about them even if multiple actions map onto the same API.
